net/modemmanager/Makefile

```diff
@@ -55,6 +54,24 @@ define Package/modemmanager/description
   Select Utilities/usb-modeswitch if needed.
 endef
 
+define Package/modemmanager-rpcd
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=RPC interface for ModemManager for rpcd
+  URL:=https://www.freedesktop.org/wiki/Software/ModemManager
+  DEPENDS:= \
+	modemmanager \
+	lua-cjson
```

Comment:
I would change this to `+lua-cjson`. I think the problem with your solution is that `modemmanager-rpcd` is only visible when `lua-cjson` is selected. I would prefer that if `modemmanager` is enabled, then `modemmanager-rpcd` can also be enabled. This will then also activate the dependencies that `modemmanager-rpcd` needs in this case `lua-cjson`.

Reply:
I agree